src/screens/WebView/HelpCenter.js

```diff
@@ -1,39 +1,39 @@
 import * as React from 'react';
-import { ActivityIndicator, SafeAreaView, StyleSheet, View } from 'react-native';
-import { WebView } from 'react-native-webview';
+import {ActivityIndicator, SafeAreaView, StyleSheet, View} from 'react-native';
+import {WebView} from 'react-native-webview';
 
 import Header from '../../components/Header';
-import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 
 const HelpCenter = ({navigation}) => (
-    <View style={styles.container}>
-      <SafeAreaView>
-      <Header title="Help Center" onPress={() => navigation.goBack()}  />
-      </SafeAreaView>
-      <WebView
-        androidHardwareAccelerationDisabled={false}
-        source={{ uri: 'https://bettersocial.org/help' }}
-        showsVerticalScrollIndicator={false}
-        startInLoadingState={true}
-        style={styles.webview}
-        renderLoading={() => (
-          <View style={styles.loading}>
-            <ActivityIndicator size="large" color="#00ADB5" />
-          </View>
-        )}
-      />
-    </View>
-  );
+  <View style={styles.container}>
+    <SafeAreaView>
+      <Header title="Help Center" onPress={() => navigation.goBack()} />
+    </SafeAreaView>
+    <WebView
+      androidHardwareAccelerationDisabled={false}
+      source={{uri: 'https://bettersocial.org/help'}}
+      showsVerticalScrollIndicator={false}
+      startInLoadingState={true}
+      style={styles.webview}
+      renderLoading={() => (
+        <View style={styles.loading}>
+          <ActivityIndicator size="large" color="#00ADB5" />
+        </View>
+      )}
+    />
+  </View>
+);
 
 export default withInteractionsManaged(React.memo(HelpCenter));
 
 const styles = StyleSheet.create({
   container: {
     // paddingHorizontal: 20,
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: '#fff'
   },
-  header: { padding: 0, paddingHorizontal: 16, paddingBottom: 8 },
-  loading: { flex: 1, position: 'relative' },
+  header: {padding: 0, paddingHorizontal: 16, paddingBottom: 8},
+  loading: {flex: 1, position: 'relative'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There are commented-out styles in your StyleSheet object (`container` and `webview`). It's generally not a good practice to leave commented-out code in your codebase as it can confuse other developers who might wonder why it's there. If these styles are no longer needed, I recommend removing them.
```diff
-  // paddingHorizontal: 20,
+  
-  // webview: {marginHorizontal: -20}
+  
```
Otherwise, if they might be needed in the future, consider adding a comment explaining why they're commented out.


<!-- This is an auto-generated comment by OSS CodeRabbit -->